### PR TITLE
Implement lepton masses in Lambda_b -> Lambda(1520)

### LINKED ISSUE #554: Update Lb -> L(1520) amplitudes accounting for lepton masses
The complete amplitudes are computed in [2003.08366](https://arxiv.org/pdf/2003.08366.pdf). Lepton masses are currently neglected in the EOS implementation. The full amplitude is needed by our LHCb colleagues.

### PR DESCRIPTION
**Pull request by mreboud**
Closes #554

### CODE REVIEW

**Comment by mreboud** [comment]
@dvandyk any idea what's going wrong here?

**Comment by mreboud** [comment]
hmm, what should we do then?

File: eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc [hunk 1]
```diff
@@ -47,7 +47,7 @@ namespace eos
 
         return g_fermi() * alpha_e() * std::sqrt(
                   1.0 / 3.0 / 2048.0 / power_of<5>(M_PI) / power_of<3>(m_Lb)
-                  * lambda_t2 * s * std::sqrt(lambda(s))
+                  * lambda_t2 * s * std::sqrt(lambda(s)) * beta_l(s)
```
Comment (dvandyk) [inline eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc:50]: This is phase space only, right?

File: eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc [hunk 2]
```diff
@@ -80,7 +80,9 @@ namespace eos
             s_plus = power_of<2>(m_Lb + m_Lstar) - s;
 
         const double alpha_s_mu = model->alpha_s(mu()); // alpha_s at the hard scale
-        const double m_b_msbar = model->m_b_msbar(mu());
+        const double
+            m_b_msbar = model->m_b_msbar(mu()),
+            m_s_msbar = model->m_s_msbar(mu());
```
Comment (dvandyk) [inline eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc:85]: I am puzzled. Wasn't ``m_s_msbar`` needed for the pseudoscalar amplitude? Ah, that only enters in the ``m_l > 0`` case, right?

File: eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc [hunk 3]
```diff
@@ -111,17 +121,25 @@ namespace eos
             HplusT5m32 = (m_Lb - m_Lstar) * sqrt(s_minus) * form_factors->f_perp32_t5(s);
 
         result.b_perp1_right =   sqrt(2.0) * norm_s * (wilson910_plus_right  * HplusVm32 - 2.0 * m_b_msbar / s * wilson7_plus  * HplusTm32);
-        result.b_perp1_left =    sqrt(2.0) * norm_s * (wilson910_plus_left   * HplusVm32 - 2.0 * m_b_msbar / s * wilson7_plus  * HplusTm32);
+        result.b_perp1_left  =   sqrt(2.0) * norm_s * (wilson910_plus_left   * HplusVm32 - 2.0 * m_b_msbar / s * wilson7_plus  * HplusTm32);
```
Comment (dvandyk) [inline eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc:124]: I think this is now inconsistent. The quark masses should enter here either as ``m_b_msbar + m_s_msbar`` or ``m_b_msbar - m_s_msbar``. You are using the ``m_s_msbar -> 0`` limit here, but not below.

File: eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc [hunk 4]
```diff
@@ -111,17 +121,25 @@ namespace eos
...
-        result.a_perp0_left =    sqrt(2.0) * norm_s * (wilson910_plus_left   * H0Vp12    - 2.0 * m_b_msbar / s * wilson7_plus  * H0Tp12);
+        result.a_perp0_left  =   sqrt(2.0) * norm_s * (wilson910_plus_left   * H0Vp12    - 2.0 * m_b_msbar / s * wilson7_plus  * H0Tp12);
         result.a_para0_right = - sqrt(2.0) * norm_s * (wilson910_minus_right * H0Ap12    + 2.0 * m_b_msbar / s * wilson7_minus * H0T5p12);
-        result.a_para0_left =  - sqrt(2.0) * norm_s * (wilson910_minus_left  * H0Ap12    + 2.0 * m_b_msbar / s * wilson7_minus * H0T5p12);
+        result.a_para0_left  = - sqrt(2.0) * norm_s * (wilson910_minus_left  * H0Ap12    + 2.0 * m_b_msbar / s * wilson7_minus * H0T5p12);
+        result.a_perpt_right =   sqrt(2.0) * norm_s * wilson10_plus * HtVp12;
+        result.a_perpt_left  = - sqrt(2.0) * norm_s * wilson10_plus * HtVp12;
+        result.a_parat_right = - sqrt(2.0) * norm_s * wilson10_minus * HtAp12;
+        result.a_parat_left  =   sqrt(2.0) * norm_s * wilson10_minus * HtAp12;
+        result.a_perpS_right =   sqrt(2.0) * norm_s / (m_b_msbar - m_s_msbar) * wilsonSP_plus_right * HtVp12;
```
Comment (dvandyk) [inline eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc:139]: "below" :smile:

**Review by dvandyk — COMMENTED** [review]
Looks mostly good, but the treatment of the strange quark mass is internally inconsistent.

File: eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc (hunk 3 above)
Comment (mreboud) [reply eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc:124]: As discussed offline this is correct due to the definition of O7

File: eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc (hunk 1 above)
Comment (mreboud) [reply eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc:50]: Yes :)

File: eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc (hunk 2 above)
Comment (mreboud) [reply eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc:85]: Exact :)

**Review by dvandyk — APPROVED** [review]
After discussing the strange quark mass offline, I agree that this is in fact consistent. My bad! Approved (pending a missing commit for the references)!

**Pull request merged**